addons/mail/static/src/new/messaging.js

```diff
@@ -136,7 +151,7 @@ export class Messaging {
     }
 
     /**
-     * todo: merge this with createThread (?)
+     * todo: merge this with Thread.insert() (?)
```

Comment:
note regarding todo: I think thread and channel should be split as much as possible